### PR TITLE
i18n(ja): Add Japanese tutorial

### PR DESCRIPTION
**Pull request by delucis**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Add Japanese tutorial.

### CODE REVIEW

File: src/content/docs/ja/tutorial/2-pages/5.mdx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+<Checklist>
+- [ ] `.css`ファイルをインポートして、グローバルなCSSスタイルを追加できる。
+</Checklist>
+</Box>
+
+### 参考
+- [Astro構文とJSXの比較](/ja/core-concepts/astro-syntax/#differences-between-astro-and-jsx)
+
+- [Astroの`<style>`タグ](/ja/guides/styling/#styling-in-astro)
```
Comment (Yan-Thomas) [inline src/content/docs/ja/tutorial/2-pages/5.mdx:150]:
```suggestion
- [Astroの`<style>`タグ](/ja/guides/styling/#astroでのスタイリング)
```

File: src/content/docs/ja/tutorial/2-pages/5.mdx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+- [AstroにおけるCSS変数](/ja/guides/styling/#css-variables)
```
Comment (Yan-Thomas) [inline src/content/docs/ja/tutorial/2-pages/5.mdx:152]:
```suggestion
- [AstroにおけるCSS変数](/ja/guides/styling/#css変数)
```

File: src/content/docs/ja/tutorial/3-components/1.mdx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+<Checklist>
+- [ ] コンテンツを再利用可能なコンポーネントにリファクタリングできる。
+- [ ] `.astro`ページに新しいコンポーネントを追加できる。
+</Checklist>
+
+</Box>
+
+### 参考 
+
+- [Astroコンポーネントの概要](/ja/core-concepts/astro-components/#component-structure)
```
Comment (Yan-Thomas) [inline src/content/docs/ja/tutorial/3-components/1.mdx:151]:
```suggestion
- [Astroコンポーネントの概要](/ja/core-concepts/astro-components/#コンポーネント構造)
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! @morinokami, @kyosuke, we are very grateful for your work towards translating this tutorial to Japanese, that's an immensurable amount of hard work we acknowledge and cherish, thank you! 💜

**Review by sarah11918 — APPROVED** [review]
I have to sneak in with a review!! (Even though I see it just merged as I am writing this!)

I am so proud, and so grateful to each and every one of you for this translation, especially @morinokami . Thank you. This is beautiful.

**Pull request merged**